packages/core/tests/unit/models/role-test.js

```diff
@@ -31,4 +31,43 @@ module('Unit | Model | role', function(hooks) {
     assert.ok(role, 'Newly created role is successfully persisted');
     assert.equal(role.id, newRole, 'Role id of new role is set as expected');
   });
+
+  test('Update records', async function(assert) {
+    assert.expect(2);
+
+    const newRole = 'new_role';
+    await Store.createRecord('role', { id: newRole }).save();
+
+    const role = await Store.findRecord('role', newRole, { reload: true });
+    role.updateOn = '2020-05-15 00:00:00.000';
```

Comment:
```suggestion
    role.updatedOn = '2020-05-15 00:00:00.000';
```
Does this test pass? I'd expect it to fail because of this typo.

Reply:
ohhh true, but it looks like I made the same typo in the assert statement, so maybe it added that property under the object.